doc/telestrat.sgml

```diff
@@ -0,0 +1,185 @@
+
+There is no tool to insert a binary in a Stratsed floppy disk. 
+
+The only way to load a binary (for Telemon 2.4) is to :
+<itemize>
+<item>remove the 20 bytes header
+<item>download osdk : http://osdk.defence-force.org/index?page=download
+<item>use Floppybuilder in OSDK to insert the binary with the tool (please read FloppyBuilder manual to insert your binary, and to start microdisc boot sector when Telestrat starts)
+</itemize>
+
+Please note also, that the binary converted into TAP file, will not produce a right stratsed file when tap2dsk and old2mfm are used. You will be in the case that Telestrat/Stratsed crashed when you do "DIR" command.
+
+If you know the Stratsed disk format, please contact the author of this doc.
+
+
+<sect>Memory layout<p>
+
+In the standard setup, cc65-generated programs use the memory from
+&dollar;0801 to &dollar;9800; so, nearly 37K of memory (including the stack) is
+available. ROM calls are possible without further precautions.
+
+
+Special locations:
+
+<descrip>
+  <tag/Stack/
+  The C runtime stack is located at &dollar;97FF (or &dollar;B3FF), and grows
+  downwards.
+
+  <tag/Heap/
+  The C heap is located at the end of the program, and grows towards the C
+  runtime stack.
+
+</descrip><p>
+
+
+
+<sect>Platform-specific header files<p>
+
+Programs containing Telestrat -specific code may use the <tt/telestrat.h/ header file.
+
+
+<sect1>Telestrat-specific functions<p>
+
+The functions listed below are special for the Telestrat. See the <url
+url="funcref.html" name="function reference"> for declaration and usage.
+
+<itemize>
+<item>explode
+<item>ping
+<item>shoot
+<item>zap
+<item>oups
+</itemize>
+
+
+<sect1>Hardware access<p>
+
+The following pseudo variables declared in the <tt/telestrat.h/ header file do allow
+access to hardware located in the address space. Some variables are
+structures; accessing the struct fields will access the chip registers.
+
+<descrip>
+
+  <tag><tt/VIA/</tag>
+  Access to the VIA (Versatile Interface Adapter) chip is available via the
+  <tt/VIA/ variable. The structure behind this variable is explained in <tt/_6522.h/.
+
+</descrip><p>
+
+
+<sect>Loadable drivers<p>
+
+<sect1>Extended memory drivers<p>
+
+No extended memory drivers are currently available for the Telestrat.
+
+
+<sect1>Joystick drivers<p>
+
+<descrip>
+
+telemon 2.4 & 3.0 manages joysticks but it had been handled yet.
+
+</descrip>
+
+
+<sect1>Mouse drivers<p>
+
+<descrip>
+
+Telestrat manages also mouse, but it had been no handled yet in this version.
+
+</descrip>
+
+<sect1>RS232 device drivers<p>
+
+<descrip>
+
+Telestrat has a RS232 port, but it's not used
+
+</descrip><
+
+<sect>Limitations<label id="limitations"><p>
+
+<sect1>Disk I/O<p>
+
+Telemon 3.0 handles fopen, fread, fclose primitives. It means that this function will crash the Telestrat because Telemon 2.4 did not have not these primitives.
```

Comment:
ok corrected here, i will push it as soon as possible.

@greg-king5 : ch376 is chip which could be use in any 8 bits cpu, because it uses a 8 bits data bus. I can share this to community, because theses routines will work in any 6502 target platform, the main problem is that there a command port and a data port on this chip which could not be the same if the ch376 is used on the others target. 
For key() converted into cget, i will have a look like TGI too